_rules/SC1-2-video-audio-description.md

```diff
@@ -115,11 +117,11 @@ A video element with an incorrect audio description.
 
 <figure id="ozplayer-1-container" class="ozplayer-container">
   <div data-controls="stack" class="ozplayer" id="ozplayer-1">
-    <video controls="controls" preload="none">
+    <video data-rule-target controls="controls" preload="none">
```

Comment:
Are we still using the data-rule-target ?
Should we repeat the note about the non-endorsement of the ozplayer here?

Reply:
Can you eleborate on the "data-rule-target" question? I did not create this implementation and am unsure about answering your question.

I repeated the note. Thank you.

Reply:
I was under the impression that we have stopped using the data-rule-target attribute to mark the element that was the target of the ACT rule.

Reply:
Good point. Fixing this now.